server/test/services/CfJsonDocumentContextTest.java

```diff
@@ -5,12 +5,37 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.testing.EqualsTester;
+import java.time.LocalDate;
+import java.time.ZoneId;
 import java.util.Optional;
+import java.util.TimeZone;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import services.applicant.predicate.JsonPathPredicate;
 
 public class CfJsonDocumentContextTest {
 
+  private static final ZoneId BEHIND_UTC_ZONE = ZoneId.of("America/New_York");
+
+  private static TimeZone origTimeZone;
+
+  @BeforeClass
+  public static void setTimeZoneToNonAmerica() {
+    origTimeZone = TimeZone.getDefault();
+    // Set the default time zone to something behind UTC. We rely on this for
+    // the date serialization / deserialization test below, where we expect
+    // everything to occur in terms of UTC rather than the system default
+    // time zone.
+    TimeZone.setDefault(TimeZone.getTimeZone(BEHIND_UTC_ZONE));
```

Comment:
I wanted to ensure the test class reset the timezone to the original value after the tests were run. This should be safe assuming we never run tests in parallel. If we were to move towards that world, if other tests relied on a pre-set time zone, we could run into problems.

If that day comes, we could instead investigate setting the system default time zone for the test runner to this for ALL tests rather than doing it on a one-off basis.

Given that we're not in that world right now, I thought it made more sense to keep the logic closer to its use.

Reply:
> Given that we're not in that world right now, I thought it made more sense to keep the logic closer to its use.

That sounds good to me too